test/e2e/cypress/integration/home.js

```diff
@@ -3,39 +3,9 @@ context('Homepage', () => {
     cy.visit('/');
   });
 
-  describe('The main links in the homepage should work (means the links are the expected ones)', () => {
-    it('should provide correct link to the Hosts page', () => {
-      cy.get('#tn-hosts-link').should('have.attr', 'href', '/hosts');
-    });
-    it('should provide correct link to the Pacemaker Clusters', () => {
-      cy.get('#tn-clusters-link').should('have.attr', 'href', '/clusters');
-    });
-    it('should provide correct link to the SAP Systems', () => {
-      cy.get('#tn-sapsystems-link').should('have.attr', 'href', '/sapsystems');
-    });
-  });
-
-  describe('The documentation links available in the homepage should work (means the links are the expected ones)', () => {
-    it('should provide correct link to the Scope document', () => {
-      cy.get('#tn-scope-link').should(
-        'have.attr',
-        'href',
-        'https://github.com/trento-project/trento/blob/main/docs/scope.md'
-      );
-    });
-    it('should provide correct link to the Readme', () => {
-      cy.get('#tn-readme-link').should(
-        'have.attr',
-        'href',
-        'https://github.com/trento-project/trento/blob/main/README.md'
-      );
-    });
-    it('should provide correct link to the Architecture document', () => {
-      cy.get('#tn-architecture-link').should(
-        'have.attr',
-        'href',
-        'https://github.com/trento-project/trento/blob/main/docs/trento-architecture.md'
-      );
+  describe('The homepage has the global health component visible', () => {
+    it('should display the global health chart', () => {
+      cy.get('#homepage-component').contains('At a glance');
```

Comment:
I think we could improve this. At least, with the currently discovered facts, check is we have 3 rows with everything on green. It is the less costly thing we can have, but I find it valuable. I think you can get it less than a half hour hehe